### DISCUSSION TITLE
Knowlg Friday's - Session 4

### DISCUSSION THREAD

**Discussion by AmiableAnil** [Sunbird-Knowlg/Community#32] [General Discussions]
Folks,

Welcome to our 4th session of Knowlg Fridays - A community connect event where we learn and grow together as a community!

[
![Knowlg Friday's Banner drawio](https://user-images.githubusercontent.com/369477/177722196-c69f92a2-6b80-4750-9ce3-263b9ecab978.png)
](https://docs.google.com/forms/d/e/1FAIpQLSfbvxHmyU3UfpeR2zhw9HkL9Zc0YdH0V8ApNW-v-_XKfKPEsQ/viewform?usp=sf_link)

**Topics:**
------

**External Fields**

- What is external fields? When to we define?
- Where external fields will get store?
- Why Cassandra? How external fields stored?
- Which objects(objectTypes) have external fields? details
- How to get object metadata with external fields?
- Limitations of external fields?

Speakers: @AmiableAnil @amitpriyadarshi

Join webinar - [Zoom link](https://us06web.zoom.us/j/86893551675?pwd=ejJ3b3NXRXl0VHI2L2NLcVB5S1dkUT09)
Meeting ID: 868 9355 1675
Passcode: 766761

@pramodkvarma, @mathewjpallan, @pkranga, @Prerna789, @rahul-kankaria, @coolbung, @aks30, @vijiurs, @ChakshuGautam, @kiranharidas187, @pritamps, @chitranshu-keshav @rahul101001000 @coolbung
@SanthoshVasabhaktula, @vrayulu, @surendrasinghs, @pallakartheekreddy, @krgauraw, @luckynvdu @reshmi-nair, @sharathkashyap, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504, @maheshkumargangula, @rhwarrier

**Comment by joffinjoy** [comment]
@AmiableAnil @vinukumar-vs Can you please post the recording link here?

**Reply by vinukumar-vs** [reply]
Topic: Knowlg Friday's - External metadata fields
Date: Jul 8, 2022 02:51 PM India

Meeting Recording:
https://us06web.zoom.us/rec/share/t2x5aBklkKY8iWB_FZvM9ZgY8hlbpFXYODrlIAfY0u-59coEuJM1FH9iHufIQVPG.qlljW4Whh0e3F20j
Passcode: +4S^XKpa


PPT presentation: [External metadata fields](https://docs.google.com/presentation/d/1j-X9HVq9rScCTVj6oGhTpCrgXEfZWPWqJx-YXp0pn80/edit?usp=sharing)

**Reply by joffinjoy** [reply]
Thanks. 

The above link doesn't work on clicks since the last underscore is omitted from the hyperlink.

So for others, either copy paste it whole or [use this](https://us06web.zoom.us/rec/share/aCwUJfwq0erxWY6-9FGFchOARGJEOyydq1fX6kYHo7FDYzwi3JDO-4MnuL99EQu7.LuFpLj-OIqmqVht_)

**Reply by vinukumar-vs** [reply]
Thanks for correcting.
